src/cli.ts

```diff
@@ -52,6 +59,12 @@ const getConfig = (env: any, args: string[]): SimulatorConfig =>
     .parse(args)
     .opts() as SimulatorConfig;
 
+let verbose: boolean;
+
 (async (): Promise<void> => {
-  return run(getConfig(process.env, process.argv));
-})().catch((err) => error(err));
+  const config = getConfig(process.env, process.argv);
+  verbose = !!config.verbose;
+  const hostSplit = config.apiHost!.split('.');
+  config.stage = hostSplit.length === 3 ? 'prod' : hostSplit[1];
```

Comment:
@coleHafner One more issue: `config.stage` should be set via `env.STAGE || 'dev'`, not parsing `hostSplit`.

Reply:
For non environment variable oriented users, this is the best way to set the stage. I updated it so `env.STAGE` overrides all and it will fall back to deriving stage from the `apiHost`.